### PR TITLE
Add https support, reduced Github Api calls and better handles env variables

### PR DESCRIPTION
**Pull request by sorsaffari**
## What is the goal of this PR?
To add https support, reduce the number of Github API calls and better handle environment variables.

## What are the changes implemented in this PR?
- listens on `443` (https), instead of `80` (http)
- stops calling the Github endpoint for fetching graknlabs member on every authenticated request, and instead fetches the list of members once every 10 minutes.
- adds dotenv dependency to handle the environment variables

### CODE REVIEW

File: service/web-server/src/server.js [hunk 1]
```diff
@@ -43,14 +48,22 @@ app.use(bodyParser.json())
...
+const { getGraknLabsMembers, getGithubUserId, getGithubUserAccessToken } = utils;
+
+let graknLabsMembers;
+const fetchGraknLabsMembersRepeatedly = async (intervalInSeconds) => {
+    graknLabsMembers = await getGraknLabsMembers(GRABL_TOKEN);
+    setInterval(async () => {
+        graknLabsMembers = await getGraknLabsMembers();
+    }, intervalInSeconds * 60 * 1000);
+}
+fetchGraknLabsMembersRepeatedly(10);
```
[the remark this answers is not in the document]
Comment (marco-scoppetta) [reply service/web-server/src/server.js]: if you change it, then we can have `intervalInSeconds` read from `config.js`? this is minor so really up to you

File: service/web-server/src/server.js [hunk 2]
```diff
@@ -17,15 +15,22 @@ const LAUNCH_EXECUTOR_SCRIPT_PATH = __dirname + '/../../launch_executor_server.s
 const DELETE_INSTANCE_SCRIPT_PATH = __dirname + '/../../delete_instance.sh';
 
 const esClient = new elasticsearch.Client({ host: ES_URI });
+utils.checkESIsRunning(esClient);
```
Comment (marco-scoppetta) [inline service/web-server/src/server.js]: do you think it looks better if we don't hide this check inside the boilerplate code that initialises variables?
something like:
```
const .....
const.....
const.....

//checks section
 -> check ElasticSearch
 -> check env variables
 -> fetch Grakn members

Comment (sorsaffari) [reply service/web-server/src/server.js]: @marco-scoppetta I'm not sure if I understand this clearly. Are you suggesting that we move the checks and fetching of members out of `utils.js` and place them at the bottom of `server.js` just before the express app starts listening on the given port?

Comment (marco-scoppetta) [reply service/web-server/src/server.js]: no necessarily, for now let's just have `const` variable declarations all up, and then let's have all the checks and the code after the variables initialisation/declaration.

Maybe in the future we can group all the initial checks together, but future!

**Pull request merged**